common/model/image.js

```diff
@@ -8,5 +8,5 @@ export type ImageType = {|
   height: number,
   alt: string,
   tasl: Tasl,
-  crops: {| [string]: ImageType |},
+  crops?: {| [string]: ImageType |},
```

Comment:
Why?

Reply:
I thought I was getting into a circular mess where a valid `{[string]: ImageType|}` _had_ to exist for every image – I didn't realise that this could just be `{}` to satisfy Flow. Have updated.